thrust/system/cuda/detail/trivial_copy.inl

```diff
@@ -66,6 +66,16 @@ template<typename System1,
   return cudaMemcpyHostToDevice;
 } // end cuda_memcpy_kind()
 
+#if defined(_WIN32) && !defined(_WIN64)
```

Comment:
I think this overload would make sense no matter the OS or bitness, right? In order to keep the conditional compilation to a minimum, could we safely remove this `#if`?